### PR TITLE
You wrote on Discord that you need more quotes. I hope you like these…

### PR DESCRIPTION
**Pull request by loui-gh**
… ones :)

Signed-off-by: Loui <Louiadmin@protonmail.com>

# Pull Request

## Description:

Added 3 new quotes.

## Related Issue:

Not related to an issue, rather a response to your request on Discord for more quotes.

Closes #XXXXX

### CODE REVIEW

File: src/config/QuoteList.ts [hunk 1]
```diff
@@ -553,4 +553,18 @@ export const QuoteList: Quote[] = [
...
-];
+  {
+    text: "Sometimes you can't see what you're learning until you come out the other side.",
+    author: "Wonder Woman",
+  },
+  {
+    text: "Life is not a series of gig lamps symmetrically arranged; \
+	life is a luminous halo, a semitransparent envelope surrounding us \
+	from the beginning of consciousness to the end.",
+    author: "Virginia Woolf",
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:564]:
```suggestion
    text: "Life is not a series of gig lamps symmetrically arranged;\nlife is a luminous halo, a semitransparent envelope surrounding us\nfrom the beginning of consciousness to the end.",
    author: "Virginia Woolf",
```
The bot is configured to parse `\n` for line breaks 🙂

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by zayidmohamedy — COMMENTED** [review]
👍

**Review by naomi-lgbt — APPROVED** [review]
Went ahead and committed the changes myself so we can get this in.

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.nhcarrigan.com)

**Pull request merged**